frontend/src/NavBar.js

```diff
@@ -51,7 +51,8 @@ export default function NavBar() {
           <Typography variant="h6" className={classes.title} component={CustomLink} to="/">
             GreySwan
           </Typography>
-          {isLoggedIn ? 
+          {/* TODO: uncomment the commented out code when deploying. */}
+          {/* {isLoggedIn ?  */}
```

Comment:
is there a way to set a flag in react?
maybe that will be easier than commenting and uncommenting each time, you can just set flag to 1 when it is local.

Reply:
Good point. I "hard coded" a flag by setting a constant called `authOn` and exporting it and I've tested locally to make sure it works. When the flag is set to "true" it passes the tests I wrote for the NavBar, and when it is set to false the other tabs are displayed in the NavBar.